forge-gui/res/cardsfolder/upcoming/intrepid_paleontologist.txt

```diff
@@ -0,0 +1,14 @@
+Name:Intrepid Paleontologist
+ManaCost:1 G
+Types:Creature Human Druid
+PT:2/2
+A:AB$ Mana | Cost$ T | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ ChangeZone | Cost$ 2 | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target card in a graveyard | ValidTgts$ Card | SpellDescription$ Exile target card from a graveyard.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Exile | Destination$ Hand | Hidden$ True | ChangeType$ Instant.,Sorcery.ExiledWithSource | ChangeNum$ 1
+S:Mode$ Continuous | MayPlay$ True | Affected$ Dinosaur.ExiledWithSource+YouOwn | AffectedZone$ Exile | Description$ You may cast Dinosaur creature spells from among cards you own exiled with Intrepid Paleontologist. If you cast a spell this way, that creature enters the battlefield with a finality counter on it. (If a creature with a finality counter on it would die, exile it instead.)
+R:Event$ Moved | ActiveZones$ Battlefield | Origin$ Stack,Exile | ValidLKI$ Card.CastSa Spell.MayPlaySource | Destination$ Battlefield | ReplaceWith$ ETBCounter
```

Comment:
Should be Static SpellCast trigger into Effect
then fix Origin + ReplacementResult